class HyasynthError(Exception):
    pass


class RemoteCallError(HyasynthError):
    pass